lib/benchmark/ips/quick.rb

```diff
@@ -0,0 +1,30 @@
+module Kernel
+  # Quickly compare multiple methods on the same method.
+  # @param obj The object to call the methods on
+  # @param methods The list of methods to call as Symbols
+  #
+  # Keyword arguments are detected as well, with the following
+  # available:
+  #
+  # @param warmup How many seconds to warm up the benchmark process
+  # @param time How many seconds to benchmark each method
+  def quick_compare(*methods)
```

Comment:
Can we use regular kwargs here?

```suggestion
  def quick_compare(*methods, warmup: nil, time: nil)
```